src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Resources/ViewsMessages.resx

```diff
@@ -243,7 +243,10 @@
   <data name="DontMissEvents" xml:space="preserve">
     <value>I nie przegap najważniejszych wydarzeń!</value>
   </data>
-  <data name="DoYouWantBeSponsor2019" xml:space="preserve">
+  <data name="DoYouWantBeSponsor2020" xml:space="preserve">
+    <value>Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2020 roku?</value>
+  </data>
+  <data name="DoYouWantToBeASponsor" xml:space="preserve">
     <value>Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2019 roku?</value>
```

Comment:
please remove that part, we don't need it anymore (it is no longer 2020 year)

Reply:
Do you mean to remove "Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2020 roku?" ?

Reply:
OK, I got your point. I removed not used  resource  DoYouWantBeSponsor - Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2019 roku) but still we need DoYouWantBeSponsor2020 which i left